lib/private/Security/Bruteforce/Throttler.php

```diff
@@ -226,6 +226,11 @@ private function isIPWhitelisted(string $ip): bool {
 	 * @return int
 	 */
 	public function getAttempts(string $ip, string $action = '', float $maxAgeHours = 12): int {
+		if ($maxAgeHours > 48) {
+			$this->logger->error('Bruteforce has to use less that 48 hours');
```

Comment:
```suggestion
			$this->logger->error('Bruteforce has to use less than 48 hours');
```

Why is this hardcoded?